snmp/datadog_checks/snmp/data/profiles/hp-ilo.yaml

```diff
@@ -0,0 +1,173 @@
+# Profile for HP iLO devices.
+# For HP iLO4 devices, a separate hp-ilo4 profile is used
+
+extends:
+  - _base.yaml
+  - _hp-base.yaml
+  - _hp-compaq-health.yaml
+  - _hp-driver-stats.yaml
+
+device:
+  vendor: "hp"
+
+sysobjectid:
+  - 1.3.6.1.4.1.232.9.4.*     # HP iLO
+
+metadata:
+  device:
+    fields:
+      product_name:
+        value: Integrated Lights-Out
+
+metrics:
+  # iLO controller metrics.
+
+  - # Power state.
+    # NOTE: unknown(1), poweredOff(2), poweredOn(3), insufficientPowerOrPowerOnDenied(4)
+    MIB: CPQSM2-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.9.2.2.32.0
+      name: cpqSm2CntlrServerPowerState
+
+  - # Status of battery.
+    # NOTE: other(1), batteryOk(2), batteryFailed(3), batteryDisconnected(4)
+    MIB: CPQSM2-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.9.2.2.5.0
+      name: cpqSm2CntlrBatteryStatus
```

Comment:
For all the status OID:
- cpqSm2CntlrServerPowerState 
- cpqSm2CntlrBatteryStatus
- cpqSm2CntlrRemoteSessionStatus
- cpqSm2CntlrInterfaceStatus

I think sending them as integer is not very user friendly. I think we should implement `contant_value_one` and `metric_tags` for scalar OID metrics.

For example:

```
- MIB: PAN-COMMON-MIB
  symbol:                                  # scalar symbol
    name: panSysHAMode
    constant_value_one: true
  metric_tags:
    - tag: mode_current
      OID: 1.3.6.1.4.1.25461.2.1.2.1.13.0  # scalar OID value
      name: panSysHAModeCurrent
      mapping:
        1: active-passive
        2: active-active
```

At the moment, constant_value_one is only supported for table columns: https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/#report-string-oids